compiler/testData/diagnostics/nativeTests/objCName.fir.kt

```diff
@@ -24,30 +24,39 @@ open class KotlinClass {
 
 @ObjCName("ObjCSubClass", "SwiftSubClass")
 class KotlinSubClass: KotlinClass() {
-    @ObjCName("objCProperty")
+    <!INAPPLICABLE_OBJC_NAME!>@ObjCName("objCProperty")<!>
     override var kotlinProperty: Int = 1
-    @ObjCName(swiftName = "swiftFunction")
-    override fun @receiver:ObjCName("objCReceiver") Int.kotlinFunction(
-        @ObjCName("objCParam") kotlinParam: Int
+    <!INAPPLICABLE_OBJC_NAME!>@ObjCName(swiftName = "swiftFunction")<!>
+    override fun <!INAPPLICABLE_OBJC_NAME!>@receiver:ObjCName("objCReceiver")<!> Int.kotlinFunction(
+        <!INAPPLICABLE_OBJC_NAME!>@ObjCName("objCParam")<!> kotlinParam: Int
     ): Int = this + kotlinParam * 2
 }
 
-@ObjCName()
-val invalidObjCName: Int = 0
+<!INVALID_OBJC_NAME!>@ObjCName()<!>
```

Comment:
`test.kt` and `test.fir.kt` must be identical modulo reported diagnostics. I'll fix it myself, so it's just a note for the future